views/items/index.pug

```diff
@@ -1,9 +1,18 @@
 extends ../layouts/authenticated-layout
 
-mixin display( nodes, depth )
+mixin display( nodes, depth, root )
```

Comment:
@jrob8577 This code was written by Rachel and Aileen . . . I've been familiarizing myself with it, and now I'm good to chat with you about it  this afternoon--come on by.